lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -65,7 +65,7 @@ BANK_ACCOUNT_START_PAGE:
     pageId: page-ipv-bank-account-start
   events:
     next:
-      targetState: PYI_ESCAPE # Todo: Need to replace with CRI.
+      targetState: CRI_BANK_ACCOUNT_J7
```

Comment:
Check with QA that doesnt disrupt exisiting tests and when we merge this is, we have tests ready to go in at the same time